### PR TITLE
Change params.py to params.json

### PR DESCRIPTION
**Pull request by Krigpl**
Replace `params.py` with `params.json` so it isn't so Python-centric and could be used by other technologies as well.

### CODE REVIEW

File: apps/dummy/resources/scripts/docker_dummytask.py [hunk 1]
```diff
@@ -1,17 +1,20 @@
 from __future__ import print_function
 
 import imp
+import json
 import os
 
-import params  # This module is generated before this script is run
+
+with open('/golem/work/params.json', 'r') as params_file:
```
Comment (maaktweluit) [inline apps/dummy/resources/scripts/docker_dummytask.py]: Should this also use the same const `WORK_DIR` as in the file above?

**Review by maaktweluit — CHANGES_REQUESTED** [review]
LGTM!

Will approve when `Golem-Verificator` is updated.

File: apps/blender/resources/scripts/docker_blendertask.py [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 import subprocess
 import sys
 from multiprocessing import cpu_count
 
-# pylint: disable=import-error
-import params  # This module is generated before this script is run
 
 BLENDER_COMMAND = "blender"
+# FIXME WORK_DIR shoudn't be hardcoded here if it's set in Dockerfile
+WORK_DIR = '/golem/work'
```
Comment (mfranciszkiewicz) [inline apps/blender/resources/scripts/docker_blendertask.py]: `WORK_DIR` is exported as an environment variable (see `docker/job.py`)

Comment (Krigpl) [reply apps/blender/resources/scripts/docker_blendertask.py]: Also it is explicitly set during container creation so this is not needed at all

**Review by maaktweluit — APPROVED** [review]

**Pull request merged**